chainer/functions/loss/softmax_cross_entropy.py

```diff
@@ -261,14 +266,32 @@ class weight ``class_weight[i]`` is actually multiplied to
             which has ``ignore_label`` as its target value, is set to ``0``.
 
     Returns:
-        Variable: A variable holding a scalar array of the cross entropy loss.
-        If ``reduce`` is ``'mean'``, it is a scalar array.
+        ~chainer.Variable: A variable holding a scalar array of the cross
+        entropy loss.  If ``reduce`` is ``'mean'``, it is a scalar array.
         If ``reduce`` is ``'no'``, the shape is same as that of ``x``.
 
     .. note::
 
        This function is differentiable only by ``x``.
 
+    .. admonition:: Example
+
+        >>> x = np.array([[-1, 0, 1, 2], [2, 0, 1, -1]]).astype('f')
+        >>> x
+        array([[-1.,  0.,  1.,  2.],
+               [ 2.,  0.,  1., -1.]], dtype=float32)
+        >>> t = np.array([3, 0]).astype('i')
+        >>> t
+        array([3, 0], dtype=int32)
+        >>> y = F.softmax_cross_entropy(x, t)
+        >>> y
+        variable(0.4401897192001343)
+        >>> log_softmax = -F.log_softmax(x)
+        >>> expected_loss = np.mean([log_softmax[row, column].data \
+for row, column in enumerate(t)])
+        >>> y.data == expected_loss
```

Comment:
How about using `y.array` instead of `y.data`?

Reply:
ok. I used `y.array`